packages/web-react/src/components/Toast/useToastStyleProps.ts

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import { useClassNamePrefix } from '../../hooks';
+import { SpiritToastProps } from '../../types';
+
+export interface ToastStyles<T> {
+  classProps: {
+    root: string;
+    queue: string;
+  };
+  props: T;
+}
+
+export function useToastStyleProps(props: SpiritToastProps): ToastStyles<SpiritToastProps> {
+  const { alignmentX, alignmentY, ...restProps } = props;
+
+  const toastClass = useClassNamePrefix('Toast');
+  const classes: string[] = [];
```

Comment:
Two indicators that are suggesting to you that you should use `reduce`:
1. you have an accumulator where you store some values
2. you are iterating over something and pushing to the accumulator
=> Array.reduce

Reply:
function refactored to use reduce